packages/nodes-base/test/nodes/TriggerHelpers.ts

```diff
@@ -70,7 +81,11 @@ export async function testTriggerNode(
 	) as INode;
 	const workflow = mock<Workflow>({ timezone: options.timezone ?? 'Europe/Berlin' });
 
-	const scheduledTaskManager = new ScheduledTaskManager(mock<InstanceSettings>());
+	const scheduledTaskManager = new ScheduledTaskManager(
+		mock<InstanceSettings>(),
+		logger as any,
```

Comment:
We'd have to make `Logger` currently at `@n8n/backend-common` available to `nodes-base` but I'd rather not add this dependency for now, only for a type. Perhaps it's the case `ScheduledTaskManager` shouldn't be getting imported outside core instead.